.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: Lint (C++, clang-tidy)
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: Install dep
+      run: |
+        sudo apt-get install -y clang-tidy libeigen3-dev
+
+    - name: Build
+      run: |
+        cd $GITHUB_WORKSPACE/cpp && mkdir build && cd build
+        cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ..
```

Comment:
Any reason not to call make here ?

Reply:
Build and test are in the circleci CI step. 

https://circleci.com/gh/hungpham2511/toppra/775?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link